src/blockchain.rs

```diff
@@ -383,8 +383,8 @@ impl FromStr for FeePriority {
 /// in fee strategies and an amount used in transactions. Implementing this trait allow to set and
 /// verify fees on transactions given a strategy and a priority.
 ///
-/// The fee to apply on transactions is carried in the [`Offer`](crate::negotiation::Offer) through
-/// a [`FeeStrategy`], in case the fee strategy allow multiple values a [`FeePriority`] is used to
+/// The fee to apply on transactions is carried in the [`Deal`](crate::trade::Deal) through a
+/// [`FeeStrategy`], in case the fee strategy allow multiple values a [`FeePriority`] is used to
```

Comment:
FeeStrategy still contains the word offer in its docstring further up on line ~172